README.md

```diff
@@ -185,8 +371,21 @@ const result = validate(value, validations)
     value: '05547-022',
     errors: [{ invalidFormat: true }]
 } */
+
```

Comment:
again, examples with valid input might not be necessary

Reply:
Removed